ccd-definition/CaseEventToComplexTypes/upload-CMO/uploadCMO.json

```diff
@@ -27,5 +27,25 @@
     "EventElementLabel": "Upload document",
     "FieldDisplayOrder": 3,
     "DisplayContext": "MANDATORY"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "SupportingEvidenceBundle",
```

Comment:
Should this not be `UploadedCaseManagementOrder`

Reply:
Also can we try explicitly hiding the Date sent and Status fields, might hide them from the CYA page for orders when you are adding more orders to a hearing